src/components/apiDelegation/reusables/OverviewPageContent/OrgDelegationActionBar/OrgDelegationActionBar.tsx

```diff
@@ -115,29 +95,48 @@ export const OrgDelegationAccordion = ({
   ));
 
   return (
-    <Accordion
+    <ActionBar
       onClick={() => {
         setOpen(!open);
       }}
       open={open}
-    >
-      <AccordionHeader
-        actions={actions}
-        subtitle={t('api_delegation.org_nr') + ' ' + organization.orgNr}
-      >
+      color={'neutral'}
+      actions={actions}
+      title={
         <div
-          className={cn(classes.orgName, {
-            [classes.accordionHeader__softDelete]: organization.isAllSoftDeleted,
+          className={cn(classes.orgDelegationActionBarTitle, {
+            [classes.actionBarText__softDelete]: organization.isAllSoftDeleted,
           })}
         >
           {organization.orgName}
         </div>
-      </AccordionHeader>
-      <AccordionContent>
-        <div className={classes.accordionContent}>
-          <List borderStyle={'dashed'}>{listItems}</List>
+      }
+      subtitle={
+        <div
+          className={cn({
```

Comment:
Ahaaaaa..
Now I get why you wanted to make them reactNodes.
Well it makes sense for our case here, even if it maybe isn't necessarily the most intuitive in a generalized component.

Reply:
I think it's the only way to gain functionality of having line through for subtitle as well. We probably'll need to the subtitle to be more flexible in the future as well, as we don't know how UX will want us to use the prop.

Reply:
True. I suppose this gives us more flexibility